src/ncbitaxon.py

```diff
@@ -187,6 +187,21 @@ def convert_node(node, label, merged, synonyms, citations):
     return "\n".join(output)
 
 
+def convert_obsolete_node(tax_id, replacement=None):
+    """Given a tax_id and an optional replacement tax_id,
+    return a Turtle string representing an obsolete taxon."""
+    output = [f"""NCBITaxon:{tax_id} a owl:Class
+        ; rdfs:label "obsolete taxon {tax_id}"^^xsd:string
```

Comment:
such a pity `delnodes.dmp`/`merged.dmp` does not contain the old label

Reply:
yeah, I even searched on the `names.dmp`, but nothing. I've checked and also ran a robot command, and it didn't complain.